src/meta/src/hummock/hummock_manager_tests.rs

```diff
@@ -72,20 +70,11 @@ async fn test_hummock_pin_unpin() {
         let pinned_versions = HummockPinnedVersion::list(env.meta_store()).await.unwrap();
         assert_eq!(pin_versions_sum(&pinned_versions), 1);
         assert_eq!(pinned_versions[0].context_id, context_id);
-        assert_eq!(pinned_versions[0].version_id.len(), 1);
-        assert_eq!(pinned_versions[0].version_id[0], version_id);
     }
 
     // unpin nonexistent target will not return error
```

Comment:
please help this fix comment (Line 75) because meaning of unpin_version has changed.

Reply:
> please help this fix comment (Line 75) because meaning of unpin_version has changed.

fixed